packages/client/hmi-client/src/services/model.ts

```diff
@@ -165,6 +165,16 @@ export async function getModelEquation(model: Model): Promise<string> {
 	return latex ?? '';
 }
 
+export const getModelVariableUnits = (model: Model) => {
```

Comment:
Chatted with @mwdchang about calling states, parameters, transitions, etc as `variables`, we settled for calling them `parts`. `variables` could be mixed with `parameters`. So `getUnitsFromModelParts`

Reply:
Done